src/main/java/mate/academy/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.Ticket;
+import mate.academy.model.User;
+import mate.academy.service.OrderService;
+import mate.academy.service.ShoppingCartService;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+    
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        if (shoppingCart.getTickets().isEmpty()) {
+            throw new RuntimeException("Your cart is empty, no orders to complete");
+        }
+        Order order = new Order();
+        List<Ticket> ticketsCopy = shoppingCart.getTickets();
+        order.setTickets(ticketsCopy);
```

Comment:
```suggestion
        List<Ticket> ticketsCopy = new ArrayList<>(shoppingCart.getTickets());
        order.setTickets(ticketsCopy);
```

if we just getTickets() it won't be a copy